nox.py

```diff
@@ -320,3 +320,12 @@ def mark_if_necessary(requirement):
         for requirement in sorted(requirements, key=lambda s: s.lower()):
             if not requirement.startswith('#'):
                 f.write(requirement)
+
+
+def session_readmegen(session):
+    session.install('-r', 'testing/requirements-dev.txt')
+
+    in_files = list(list_files('.', 'README.rst.in'))
```

Comment:
`.in` indicates that the file is used to generate the file of the same name before it, so `README.rst.in` -> `README.rst`, whereas `README.yaml.in` -> `README.yaml`.